### DISCUSSION TITLE
Database size 2-3 times compared to actual MySQL volume with 1.22.0

### DISCUSSION THREAD

**Discussion by vdbhb59** [FreshRSS/FreshRSS#5778] [Q&A] [answered]
**Describe the bug**
I check my hosted database size almost daily. Till I upgraded to 1.22.0, my DB size was 2.18GB, with approx 970 Users and the moment after the upgrade the DB size went up to 5.76GB.

Actual DB size on my server was still 2.18GB.

Before creating this ticket, I optimized once again. My SQL is 1.9GB while the instance archive page shows 5.1GB.

**To Reproduce**
1. Go to archive under settings
2. Click on optimize
3. See the resulting optimized data size
4. Cross check with actual MySQL size

**Expected behavior**
Both the instance and actual DB size should be in sync.

**Screenshots**
N.A.

**Environment information (please complete the following information):**
- Device: Irrespective
- OS: CentOS 7
- Browser: Irrespective
- FreshRSS version: 1.22.0
- Database version: Mysql 5.7.43
- PHP version: 8.0.29
- Installation type: Manual ZIP

**Comment by Alkarex** [comment, marked as answer]
Hello,
There has been a change in the calculation with https://github.com/FreshRSS/FreshRSS/pull/5655, which now includes `DATA_FREE` in the calculation https://mariadb.com/kb/en/information-schema-tables-table/
You could try to compare with the size of your volume on disk.

**Comment by vdbhb59** [comment]
Ahh, that explains the reason. Will check if it matches as well. Thanks again. :)

**Reply by vdbhb59** [reply]
I checked and yes, the size is correct over MySQL and the actual DB, but just that FRSS shows the buffer/cache as well.

**Reply by Alkarex** [reply]
Excellent; thanks for the feedback 👍🏻
